components/admin_console/admin_definition.jsx

```diff
@@ -3164,6 +3164,15 @@ export default {
                         help_text_default: 'When true, webhooks, slash commands and other integrations, such as [Zapier](!https://docs.mattermost.com/integrations/zapier.html), will be allowed to change the profile picture they post with. Note: Combined with allowing integrations to override usernames, users may be able to perform phishing attacks by attempting to impersonate other users.',
                         help_text_markdown: true,
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'ServiceSettings.CreateBotAccounts',
+                        label: t('admin.service.createBotTitle'),
+                        label_default: 'Enable Bot Accounts',
+                        help_text: t('admin.service.enableBotAccounts'),
```

Comment:
The keys here should have the same prefix:
* `admin.service.createBotTitle`
* `admin.service.createBot`

Right now there's a mix of `createBot` and `enableBot`, which seems confusing.